lib/src/getter_utils.dart

```diff
@@ -15,10 +16,9 @@ abstract class GetterUtils {
         isUtc: true);
   }
 
-  static RedditorRef redditorRefOrNull(Reddit reddit, String redditor) {
-    if (redditor == null) {
-      return null;
-    }
-    return reddit.redditor(redditor);
-  }
+  static RedditorRef redditorRefOrNull(Reddit reddit, String redditor) =>
+      (redditor != null) ? reddit.redditor(redditor) : null;
```

Comment:
`(redditor == null) ? null : reddit.redditor(redditor);`

Maybe it's a personal thing but I really don't like negations, just mildly extra brain power that I don't want to use up while reading code, but you don't have to change it. 😞

Reply:
Actually on second thought, I don't think it makes a difference here anyway!